pynsee/localdata/get_geo_list.py

```diff
@@ -4,13 +4,18 @@
 import os
 import pandas as pd
 from tqdm import trange
+import numpy as np
+from requests.exceptions import RequestException

```

Comment:
this should be removed together with most changes in this file

Reply:
still true

Reply:
Same here, this should be aligned on the current master.